.dependabot/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 1
+update_configs:
+  - package_manager: "ruby:bundler"
+    directory: "/"
+    update_schedule: "live"
+    target_branch: "development"
+
+    default_reviewers:
+      - "echarrod"
+      - "davidgrayston"
+      - "emmas-yoti"
+      - "MrBurtyyy"
```

Comment:
Do we need to exclude this file in in https://github.com/getyoti/yoti-ruby-sdk/blob/master/yoti.gemspec#L21 ?